static/app/views/eventsV2/resultsChart.tsx

```diff
@@ -99,6 +99,11 @@ class ResultsChart extends Component<ResultsChartProps> {
           )
         : eventView.interval;
 
+    const seriesLabels = yAxisValue.map(stripEquationPrefix);
+    const disableableSeries = [
+      ...seriesLabels,
+      ...seriesLabels.map(label => `previous ${label}`),
```

Comment:
Aside, but this `previous ..` relies on the strings being correct on both places -> should likely be changed into a function used both here, and wherever the previous string is becoming the series in the first place.